skopt/callbacks.py

```diff
@@ -304,3 +312,15 @@ def __call__(self, res):
             The optimization as a OptimizeResult object.
         """
         dump(res, self.checkpoint_path, **self.dump_options)
+
+    def load_previous_evaluations(self):
+        """
+        Loads from disk previously evaluated points.
+        Returns
+        -------
+        X, y : np.ndarray, np.ndarray
```

Comment:
I personally would prefer the method name shorter and the return value a dict, such that can be used more simply:
```py
gp_minimize(..., **checkpoint_callback.load())
```